docs/shop_administration/manage_products.md

```diff
@@ -39,9 +39,19 @@ When adding attributes to product types, you can add both individual attributes,
 
 ![Adding attributes to a product type](img/adding_attributes.png)
 
+### Attributes and variants
+
+Product attributes are the basis for creating variants.
+
+For every attribute, you can select **Used for product variants**.
+Attributes that have this option are used to [automatically generate product variants](product_variants_and_assets.md#generating-variants).
+
 ## Product availability and stock
 
-To control a product's availability and the available stock, use the product's **Availability** tab.
+To control a product's availability and the available stock, use the product's or product variant's **Availability** tab.
+
+If a product can have variants, you must set availability for each variant separately.
+If a product cannot have variants, you can set availability for the whole product.
```

Comment:
```suggestion
To control a product's availability and the available stock, use the product's **Availability** tab.
If a product can have variants, you must set availability for each variant separately.
```
In this case, is it totally clear that without variants availability is set for base products? We had a problem with clarity in the corresponding dev-doc PR.

Reply:
For me it is more obvious like that, but if we had problems with it, I'm ok with 'don't wash your baby in the washing machine' type of instruction :)